### ISSUE TITLE
attacking other players

### ISSUE THREAD

**Issue by saviourom** [Alex-the-666/DomesticationInnovation#227]
Cannot attack other players without holding shift when using hardcore revive mod and epic fight. 1.20.1

**Comment by Insane96** [comment]
I don't have those mods and experience the same problem.

The only thing context I have is that we're in the same `/team`

**Comment by VenomiXeD** [comment]
After peeking around in the code, i can confirm this behavior is hardcoded and intended. As there's no config for it.
https://github.com/AlexModGuy/DomesticationInnovation/blob/7b0fd3454a906e35013d70c99d3115501ec1a972/src/main/java/com/github/alexthe668/domesticationinnovation/server/CommonProxy.java#L712-L723

**Comment by Insane96** [comment]
Definitely not intended. The player cannot be tamed

**Comment by VenomiXeD** [comment]
Yes and no, There's a mixin that changes the return value of isAlliedTo, and that's used by isPetOf. Intended or not - does not really matter. But I wish this was not the case.

I forked the project and made a commit that is a very band-aid solution but it should work in theory.
https://github.com/AlexModGuy/DomesticationInnovation/commit/f5f5a983f320c549e66447174e186add9fc8019d
